web/templates/navbar_alerts/time_zone_update_offer.hbs

```diff
@@ -0,0 +1,10 @@
+<div data-step="1">
+    {{t "Your computer's time zone has changed. Update your profile's time zone to {browser_time_zone}?" }}
```

Comment:
I think this can just be "your time zone".

Reply:
Yeah, "your profile time zone" could be OK, but probably your briefer version is better.

Reply:
Changed to "your time zone"

Reply:
Changed to "your time zone"